src/plugins/dataflow-tool/components/dataflow-program.tsx

```diff
@@ -657,12 +660,13 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
         }
       });
     }
-    if (!hasRelay && !hasDataStorage && !hasDemoOutput) {
+    if (!hasRelay && !hasDataStorage && !hasDemoOutput &&!hasLiveOutput) {
```

Comment:
On a broader note, these tests are getting out of hand. Individual nodes should be able to indicate whether they are output blocks (or any other properties of interest) so that this code can handle nodes generically. Maybe now isn't the time for that kind of refactor, but we can't keep going down this path of adding special-case code for every new kind of block.